store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9925,8 +9925,80 @@ TODO: delete them permanently from here
+  <globalConfigValue>8</globalConfigValue>
+  <desc>The number of days backups will be kept. Backup Scheduler deletes backups older than the set value.</desc>
+</attr>
+
+<attr id="4012" name="zimbraBackupBlobsCompressType" type="enum" value="zip,zipStore,noZip" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+  <globalConfigValue>zip</globalConfigValue>
+  <desc>Blobs inside a backup are compressed by default in zip format.
+        zip - blobs are backed up into zip files with compression (default).
+        zipStore - blobs are backed up into zip files without compression.
+        noZip - blobs are backed up as individual files rather in zip files.
+  </desc>
+</attr>
+
+<attr id="4013" name="zimbraBackupSyncEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+  <globalConfigValue>FALSE</globalConfigValue>
+  <desc>If true, the full backup runs synchronously.
+        If false, full backup runs in a separate thread and the soap call returns with the backup label.
+  </desc>
+</attr>
+
+<attr id="4014" name="zimbraMailReportEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+  <globalConfigValue>TRUE</globalConfigValue>
+  <desc>If true, sends an email report to the admin user. Admins can disable the email notification by setting it to FALSE.</desc>
+</attr>
+
+<attr id="4015" name="zimbraBackupStartDay" type="string" max="32" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+  <globalConfigValue>6</globalConfigValue>
+  <desc>The day of the week on which the full backup starts.
+        When a day is select their relative value is set on the crontab. Default starts from Saturday
+        0-7 or names allowed (Sun, Wed), with both 0 and 7 representing Sunday. Monday to Saturday (1-6)
+  </desc>
+</attr>
+
+<attr id="4016" name="zimbraBackupObjectLevelEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+  <globalConfigValue>TRUE</globalConfigValue>
+  <desc>If true, --accounts option will be set to zimbraBackupConfigEnabled to zmschedulebackup command. Enables cos,domain,server,account level backup.
+        If false, --accounts option will be set to all accounts to zmschedulebackup command. Backups all accounts.
+  </desc>
+</attr>
+
+<attr id="4017" name="zimbraBackupCrontabConfig" type="string" cardinality="multi" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+  <globalConfigValue>FALSE</globalConfigValue>
+  <desc>Configures crontab schedule. Full, incremental and delete cron schedules can be set using this attribute
+        f '0 1 * * 6' 'i 0 1 * * 0-5' d 1m '0 0 * * *' values can be passed
```

Comment:
Does this mean set multiple settings (`f, i, d` in this example) as a single value? 
What "multiple" values can/should be set? Set `f xxx`, `i xxx` and `d xxx` separately?
Please describe it more user friendly.